### DISCUSSION TITLE
Official UI

### DISCUSSION THREAD

**Discussion by morigs** [GreptimeTeam/greptimedb#583] [Ideas]
Are there any plans/work on the official ui for greptime?
So that it would be possible to check cluster status, list of tables, statistics etc

**Comment by alili** [comment]
Of course! 

We hope to provide developers with some products out of the box. 
starting with a simple query and table and chart display. We will use Greptimedb as the running data store, which seems to be a good showcase. 

If all goes well, a very initial but working version should be available next month.

**Comment by sarahlau0415** [comment]
hi Igor, there is a dashboard available now~~https://github.com/GreptimeTeam/dashboard
